### PR TITLE
Update Design Assets

### PR DESCRIPTION
**Pull request by thinkocapo**
## DONE
- Header Logo png added
- Texts - applied font Rubik Regular
- Buttons - applied font Rubik Medium
- Buttons - updated the purple
- Buttons - rounded the corners
- Buttons - added black border at 20% opacity
- Buttons - lowercased the fonts on them
- Texts/Buttons - reduced text header size so it's smaller in proportion to the buttom, per how it looks in Figma
- Margins - fixed vertical spacings between headers+buttons
- Action Bar - removed 'android' action bar at top, manifest, see comment. 

and
by Manoel:
- Left alignment of Header texts to the buttons, like in Figma
- Add Shadow to buttons, like in Figma, android:elevation="2dp" https://developer.android.com/training/material/shadows-clipping#Shadows
- 'Fixed' positioned header. Scrollable buttons underneath. widget for ScrollView, w/ components inside
- autoupload the debug symbols. remove the 'upload_debug_files' command from Makefile.
- replace 'grey' in status bar with Darker Purple #562e7d or Black

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,59 +6,39 @@
     android:layout_height="match_parent"
     tools:context="com.example.vu.android.MainActivity">
 
+<!--  04/03/2020 Couldn't get these to work, they belong on a <activity> element but we're using <androiconstraintlayout> -->
+<!-- android:theme=-->
+    <!--    @android:style/Theme.Holo.NoActionBar.Fullscreen -->
+    <!--    @android:style/Theme.Black.NoTitleBar -->
+    <!--    @android:style/Theme.Black.NoTitleBar.Fullscreen-->
+    <!--    android:theme="@android:style/Theme.NoTitleBar"-->
```
Comment (marandaneto) [inline app/src/main/res/layout/activity_main.xml:14]: `android:theme` should be set in the `AndroidManifest file, `application` tag

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -16,8 +16,10 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        android:theme="@style/Theme.AppCompat.NoActionBar"
```
Comment (thinkocapo) [inline app/src/main/AndroidManifest.xml:19]: @marandaneto here is how I removed action bar.

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (marandaneto) [reply app/src/main/res/layout/activity_main.xml:14]: you can remove and resolve this

File: README.md [hunk 3]
```diff
@@ -46,7 +46,11 @@ Additional documentation:
     ```
 
 4. Put your Sentry DSN key in `AndroidManifest.xml`
-5. `make all`
+5. Put your AUTH Token in sentry.properties
+
+6. `make all`
+
+7. Android Studio install ANdroid NDK in Preferences > System & Behavior > System Settings > Android SDK > SDK Tools > select NDK for download
```
Comment (marandaneto) [inline README.md:53]:
```suggestion
7. Android Studio install Android NDK in Preferences > System & Behavior > System Settings > Android SDK > SDK Tools > select NDK for download
```

**Comment by thinkocapo** [comment]
Manoel merged into this PR and I tested it. Works.

**Pull request merged**